### PR TITLE
Use postgres port on startup

### PR DESCRIPTION
**Pull request by glombiewski**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

- Added postgres port option to server startup.

### CODE REVIEW

**Comment by glombiewski** [comment]
I did not see any server startup tests, but if there are any, let me know and I'll add a test for this case.

**Comment by glombiewski** [comment]
[Postgres Config Port wird nicht verwendet](https://trello.com/c/CFJQsBcG/1456-postgres-config-port-wird-nicht-verwendet)

File: services/src/pro/server.rs [hunk 1]
```diff
@@ -222,6 +222,7 @@ async fn start_postgres(
             .password(&db_config.password)
             .host(&db_config.host)
             .dbname(&db_config.database)
+            .port(db_config.port)
```
Comment (michaelmattig) [inline services/src/pro/server.rs:225]: can you also add this to the non-pro version: https://github.com/glombiewski/geoengine/blob/814d06609161cbdc75cc96bc3a58f1112d068630/services/src/server.rs#L53

**Comment by michaelmattig** [comment]
Startup tests are here; https://github.com/glombiewski/geoengine/blob/814d06609161cbdc75cc96bc3a58f1112d068630/services/tests/startup.rs#L49

File: services/src/pro/server.rs (hunk 1 above)
Comment (glombiewski) [reply services/src/pro/server.rs:225]: 9cd7f7ac6e175ab4a0b9af46e88a8f798e7c8fc3

**Review by michaelmattig — APPROVED** [review]

**Pull request merged**